docs/using/tutorial/Vagrantfile.template

```diff
@@ -10,6 +10,41 @@ VAGRANTFILE_API_VERSION = "2"
 
 ENV['VAGRANT_DEFAULT_PROVIDER'] = 'virtualbox'
 
+$bootstrap_common = <<EOF
+mkdir -p /etc/flocker
+cp /home/vagrant/credentials/cluster.crt /etc/flocker/cluster.crt
+EOF
+
+$bootstrap_control = <<EOF
+cp /home/vagrant/credentials/control-172.16.255.250.crt /etc/flocker/control-service.crt
+cp /home/vagrant/credentials/control-172.16.255.250.key /etc/flocker/control-service.key
+systemctl enable flocker-control
+systemctl start flocker-control
+EOF
+
+$bootstrap_node1_cert = <<EOF
```

Comment:
:question: Perhaps `bootstrap_node1_cert` and `bootstrap_node2_cert` can be combined and script arguments can be used - see http://docs.vagrantup.com/v2/provisioning/shell.html.